Formula/asdf.rb

```diff
@@ -27,6 +28,20 @@ def install
     prefix.install Dir["*"]
   end
 
+  def caveats
+    <<~EOS
+      In order to enable asdf, add the following to your .bash_profile or .zshrc
```

Comment:
What does "enable" mean in this context? Because it works in the test below

Reply:
Executing the binary itself works, though for `asdf` to appear on your shell path and be used as intended, the `asdf.sh` script needs to be added to your shell config file.

We have in depth instructions on our site for the combination of OSs, Shell & installation method as there are many. It's probably better we just link to the docs instead of duplicating the instructions here.

Reply:
Anything that is definable in documentation is not supposed to be in caveats. Those are only for homebrew specific quirks or issues.

Reply:
Right, but the software itself isn't usable until external actions are taken, surely that's not a standard expectation of Homebrew users?

Reply:
That's why they're supposed to read the documentation of the tool they install.

Reply:
Yes, I would love that to be the case, but in reality, people need as more help. A simple print statement here saying: 

```
to complete configuration of asdf, see the docs at https://asdf-vm.com/#/core-manage-asdf-vm
```

would save me triaging issues.

When I tell macOS colleagues about a new app or tool, they simply run `brew install <app_name>` and never check the docs site of the tool. Brew's zero hassle installs, handling all dependencies etc has conditioned them this way.

Reply:
We do have something similar for z: https://github.com/Homebrew/homebrew-core/blob/master/Formula/z.rb#L18-L23

Reply:
Should I show the message @jthegedus suggest?

> to complete configuration of asdf, see the docs at https://asdf-vm.com/#/core-manage-asdf-vm

Reply:
I would prefer that than rehashing the various combinations here.

Reply:
If it's in upstream documentation it shouldn't be in the caveats. And since "read the docs here" isn't homebrew specific that shouldn't be in caveats either. (Other formula having something doesn't mean it's allowed)

Reply:
The caveat here is that there are steps after brew install. This is unlike the 99% of brew installed applications. I believe this fits well within the definition of caveat.

Reply:
I've fixed my implementations to @jthegedus 's idea
Before doing my work, I looked for the right place to show some instructions on Homebrew Formula, then I found `caveats` is most suitable
`asdf` has an attribute to be configured by users, I believe this is one of core of `asdf`
Showing instructions is natural for this tool